frontend/conf/touchpoint.PROD.conf

```diff
@@ -0,0 +1,64 @@
+# ***NO PRIVATE CREDENTIALS IN THIS FILE *** - use membership-keys.conf in S3 for private data
+touchpoint.backend.environments {
+    PROD {
+        stripe {
+            api.key {
+                secret = ""
+                public = "pk_live_2O6zPMHXNs2AGea4bAmq5R7Z"
+            }
+        }
+
+        salesforce {
+            consumer {
+                key = ""
+                secret = ""
+            }
+            api {
+                url="https://emea.salesforce.com"
+                username=""
+                password=""
+                token=""
+            }
+        }
+
+        zuora {
+            api {
+                url="https://api.zuora.com/apps/services/a/58.0"
+                restUrl="https://api.zuora.com/rest/v1"
+                username=""
+                password=""
+            }
+            ratePlanIds {
+                friend="2c92a0f9479fb46d0147d0155c6f558b"
+                staff="2c92a0f949efde7c0149f1f18162178e"
+                supporter {
+                    monthly="2c92a0f94c547592014c69f5b0ff4f7e"
+                    yearly="2c92a0fb4c5481db014c69f4a1e03bbd"
+                }
+                partner {
+                    monthly="2c92a0fb4c5481dc014c69f95fce7240"
+                    yearly="2c92a0f94c54758b014c69f813bd39ec"
+                }
+                patron {
+                    monthly="2c92a0fb4c5481db014c69fb9118704b"
+                    yearly="2c92a0f94c547592014c69fb0c4274fc"
+                }
+                legacy {
```

Comment:
This hard-coding of legacy rateplanids makes me sad - I can understand the need for whitelisting the currently-on-sale rateplans - we don't wwant to sell the wrong thing - but I believe it should be possible to construct a model of 'legacy' tierplans from looking at all rateplans under products in the Product catalog that have a `Tier__c` field set. The billing period is also available from this information.


```
"id" : "2c92a0f9479fb46d0147d0155a3d5548",
    "sku" : "SKU-00000006",
    "name" : "Partner",
    "description" : "This is the description of the partner membership.",
    "ProductType__c" : "Membership",
    "ProductEnabled__c" : "True",
    "Entitlements__c" : "Events:PARTNER",
    "AcquisitionProfile__c" : "Paid",
    "Tier__c" : "Partner",
    "ProductLevel__c" : "200",
    "productRatePlans" : [ {
      "id" : "2c92a0f9479fb46d0147d0155cb15596",
      "status" : "Expired",
      "name" : "Partner - annual",
      "description" : "",
      "effectiveStartDate" : "2014-01-01",
      "effectiveEndDate" : "2015-06-23",
      "RatePlanType__c" : "Base",
      "DefaultTerm__c" : null,
      "TermType__c" : null,
      "PromotionCode__c" : null,
      "productRatePlanCharges" : [ {
        "id" : "2c92a0fb479fbd5d0147d01563184625",
        "name" : "Partner Membership - Annual",
        "type" : "Recurring",
        "model" : "FlatFee",
        "pricingSummary" : [ "GBP135" ],
        "billingDay" : "ChargeTriggerDay",
        "billingPeriod" : "Annual",
        "taxable" : true,
        "taxCode" : "Global Tax",
        "taxMode" : "TaxInclusive"
      } ]
    }, {
      "id" : "2c92a0f9479fb46d0147d0155ca15595",
      "status" : "Expired",
      "name" : "Partner - monthly",
      "description" : "",
      "effectiveStartDate" : "2014-01-01",
      "effectiveEndDate" : "2015-05-02",
      "RatePlanType__c" : "Base",
      "DefaultTerm__c" : null,
      "TermType__c" : null,
      "PromotionCode__c" : null,
      "productRatePlanCharges" : [ {
        "id" : "2c92a0fb479fbd5d0147d01562c9461a",
        "name" : "Partner Membership - Monthly",
        "type" : "Recurring",
        "model" : "FlatFee",
        "pricingSummary" : [ "GBP15" ],
        "billingDay" : "ChargeTriggerDay",
        "billingPeriod" : "Month",
        "taxable" : true,
        "taxCode" : "Global Tax",
        "taxMode" : "TaxInclusive"
      } ]
    },
```